content/en/docs/compatibility-matrix.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Product compatibility matrix
+description: "Information about what Armory Enterprise supports."
+---
+
+<!-- If you don't want to make markdown tables manually, use something like https://www.tablesgenerator.com/markdown_tables# -->
+
+This page describes the tools that Spinnaker integrates with.
+
+## Authentication
+
+The following table lists the authentication protocols that Spinnaker supports:
+
+| Identity provider     | Spinnaker version | Note                                        |
+|-----------------------|-------------------|---------------------------------------------|
+| SAML                  | All supported versions      |                                             |
+| OAuth 2.0             | All supported versions      | Can use Azure, GitHub, Google, Oracle Cloud |
```

Comment:
Actually we have a lot more options than Azure, GitHub, Google. We also have docs for Okta, OneLogin, Auth0. But generally speaking I think anything that is OAuth 2.0/OIDC compliant works.